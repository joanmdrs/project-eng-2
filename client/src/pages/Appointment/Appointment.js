import "./Appointment.css"
import React from "react";
import Nav from "../../components/Nav/Nav";
import 'bootstrap/dist/css/bootstrap.min.css';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import  { faFileWaveform } from '@fortawesome/free-solid-svg-icons'
import {
    Form,
    FormGroup,
    Label,
    Input,
    Row,
    Col,
    Button
} from 'reactstrap'

export default function Appointment(){

    return (
        <div className="container-appointment">
            <Nav />
            <div className="box-appointment">

                <div className="box-header-appointment">
                    <div className="box-icon-and-title">
                        <FontAwesomeIcon className="icon" icon={faFileWaveform} /> 
                        <h3> Appointments </h3>
                    </div>
                    <div className="box-list-functions">
                        <p>
                            Appointment registration: Include, Search, Change, Delete and List
                        </p>
                    </div>
                </div>
                <Form className="form-appointment">
                    <Row>
                        <Col md={8}>
                            <FormGroup>
                                <Label for="title">
                                    Title
                                </Label>
                                <Input 
                                    id="title"
                                    type="text" 
                                    placeholder="title">
                                </Input>
                            </FormGroup>
                        </Col>
                        <Col md={4}>
                            <FormGroup>
                                <Label for="appointment-date">
                                        Date
                                </Label>
                                <Input 
                                    id="appointment-date"
                                    type="date" 
                                    placeholder="appointment date" >
                                </Input>
                            </FormGroup>
                        </Col>
                    </Row>
                    <Row>
                        <Col md={2}>
                            <FormGroup>
                                <Label for="patient-id">
                                    Patient id
                                </Label>
                                <Input 
                                    id="patient-id"
                                    type="number" 
                                    placeholder="patient id">
                                </Input>
                            </FormGroup>
                        </Col>
                        <Col md={10}>
                            <FormGroup>
                                <Label for="patient-name">
                                    Patient name
                                </Label>
                                <Input
                                    id="patient-name"
                                    type="text" 
                                    placeholder="patient name">
                                </Input>
                            </FormGroup>
                        </Col>
                    </Row>
                    <Row>
                        <Col md={2}>
                            <FormGroup>
                                <Label for="doctor-id">
                                    Doctor id
                                </Label>
                                <Input 
                                    id="doctor-id"
                                    type="number" 
                                    placeholder="doctor id">
                                </Input>
                            </FormGroup>
                        </Col>
                        <Col md={10}>
                            <FormGroup>
                                <Label for="doctor-name">
                                    Doctor name
                                </Label>
                                <Input
                                    id="doctor-name"
                                    type="text" 
                                    placeholder="doctor name">
                                </Input>
                            </FormGroup>
                        </Col>
                    </Row>
                    <Row >
                        <Col md={8}>
                            <FormGroup>
                                <Label for="appointment-description">
                                Description
                                </Label>
                                <Input 
                                    id="appointment-description"
                                    type="textarea" 
                                    placeholder="description"
                                    className="input-textarea" >
                                </Input>
                            </FormGroup>
                        </Col>
                        <Col className="col-btn-save-cancel" md={4}>
                            <Button type="button" className="btn-save btn-success">
                                Save
                            </Button>
                            <Button  type="submit" className="btn-cancel btn-danger">
                                Cancel
                            </Button>
                        </Col>
                    </Row>

               </Form>
            </div>
        </div>
    )
} 