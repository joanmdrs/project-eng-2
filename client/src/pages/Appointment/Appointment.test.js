import renderer from 'react-test-renderer';
import Appointment from "./Appointment";

it('Testando a renderização da page Appointment', () => {
    const component = renderer.create(Appointment);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot(); 
}) 