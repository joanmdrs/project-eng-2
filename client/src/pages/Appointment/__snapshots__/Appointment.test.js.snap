// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando a renderização da page Appointment 1`] = `null`;
